app/views/early_years_payment/provider/landing_page.html.erb

```diff
@@ -0,0 +1,139 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t("early_years_payment_provider.landing_page.title") %></h1>
+
+    <p class="govuk-body-l">
+      Use this service to make an early years financial incentive payment claim
+      for an employee.
+    </p>
+
+    <h2 class="govuk-heading-l govuk-!-margin-top-7">Eligibility</h2>
+
+    <p class="govuk-body">
+      To make a claim for an employee, they must:
+    </p>
+
+    <%= govuk_list [
+      "have applied for a job with an incentive attached",
+      "have started their job after 15 May 2024",
+      "spend most of the time in their job (70% or more) working directly with children - there is no minimum number of hours and applies to all EYFS part and full time roles"
+    ], type: :bullet %>
+
+    <p class="govuk-body">
+      Working directly with children can involve any of the following:
+    </p>
+
+    <%= govuk_list [
+      "working in the room (for example this can include playrooms, baserooms or classrooms)",
+      "guiding, supporting, and interacting with children in their learning and development",
+      "giving hands-on care, including health and hygiene"
+    ], type: :bullet %>
+
+    <h3 class="govuk-heading-m">Other conditions for the payment</h3>
+
+    <p class="govuk-body">
+      The purpose of this incentive scheme is to increase the number of people
+      working with children in permanent roles, within the early years sector.
+    </p>
+
+    <p class="govuk-body">
+      In the six months before starting their current job, your employee must
+      not have had a permanent job working directly with children in an early
+      years setting.
+    </p>
+
+    <p class="govuk-body">
+      If they did work in early years in the 6 months before starting their
+      current job, you can claim if this was in the following types of
+      employment:
+    </p>
+
+    <%= govuk_list [
+      "voluntary or unpaid work",
+      "casual and temporary"
+    ], type: :bullet %>
+
+    <p class="govuk-body">
+      Casual or temporary work could include:
+    </p>
+
+    <%= govuk_list [
+      "zero-hour contracts",
+      "guaranteed minimum hour contracts",
+      "self-employed or freelance contracts",
+      "fixed term contracts (FTCs)",
+      "agency work and apprenticeship jobs"
+    ], type: :bullet %>
+
+    <p class="govuk-body">
+      You can find more information about employment contracts on the
+      <%= govuk_link_to("ACAS website", "#") %>.
```

Comment:
@cwadsworthdfe - could you let me know where this should link to?

Reply:
Its this one: 
https://www.acas.org.uk/employment-contracts

Reply:
thanks! updated